Homework-7/Validations/CountryCheckAttribute.cs

```diff
@@ -0,0 +1,23 @@
+﻿
+using System.ComponentModel.DataAnnotations;
+using static Homework_7.Enum.Country;
+
+namespace Homework_7.Validations
+{
+    public class CountryCheckAttribute : ValidationAttribute
+    {
+        protected override ValidationResult IsValid(object value, ValidationContext validationContext)
```

Comment:
you can directly over ride this function as well

public override bool IsValid(object? value)

eg:

```
public class PhoneNumberValidationAttribute: ValidationAttribute
	{
        public override bool IsValid(object? value)
        {
            string? phoneNumber = value?.ToString();

            if (string.IsNullOrWhiteSpace(phoneNumber))
            {
                ErrorMessage = "Phone number required";
                return false;
            }

            string pattern = @"^\+\d{10,15}$"; // Regex pattern for "+1234567890"

            if (!Regex.IsMatch(phoneNumber, pattern))
            {
                ErrorMessage = "Phone number must be of valid format (e.g., \"+1234567890\")";
                return false;
            }

            return true;
        }
    }
```